### ISSUE TITLE
Support for Azure DevOps style link/anchor?

### ISSUE THREAD

**Issue by cdudley-oncore** [lijunle/Vsxmd#90]
Hi,

This is a very nice plugin and it saves us a ton of work. One thing I've noticed is that links/anchors work well for GitHub, but are not in the form supported by Aznre DevOps markdown wikis.

For example, this works on GitHub:
[link text](#Anchor-I-Want-To-Link-To)
<a name='T-Anchor-I-Want-To-Link-To'></a>

This is what the Azure DevOps wiki expects:
[link text](#Heading-I-Want-To-Link-To)
## Heading-I-Want-To-Link-To

Is it possible to include this option?

Thanks!

**Comment by Nefariis** [comment]
Yeah I am running into this as well - I would love to get this working on azure.
